conda/pytorch-cuda90-0.3.1/meta.yaml

```diff
@@ -8,16 +8,19 @@ source:
 
 requirements:
   build:
-    - python
+    - python {{PY_VER}}*,>=2.7,<3.3|>=3.4
     - numpy 1.11.*
     - setuptools
     - pyyaml
     - cffi
     - mkl
-    - magma-cuda90 # [not osx]
+    - typing
+    - magma-cuda90 # [not osx and not win]
+    - ninja # [win]
```

Comment:
Why is ninja limited to windows?

Reply:
ninja isn't used in Linux builds currently